lib/shopify_app/controller_concerns/payload_verification.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module ShopifyApp
+  module PayloadVerification
+    extend ActiveSupport::Concern
+
+    def shopify_hmac
+      request.headers['HTTP_X_SHOPIFY_HMAC_SHA256']
+    end
+
+    def hmac_valid?(data)
+      secrets = [ShopifyApp.configuration.secret, ShopifyApp.configuration.old_secret].reject(&:blank?)
+
+      secrets.any? do |secret|
+        digest = OpenSSL::Digest.new('sha256')
+        ActiveSupport::SecurityUtils.secure_compare(
+          # request.headers['HTTP_X_SHOPIFY_HMAC_SHA256'],
```

Comment:
Was this accidentally left in?

Reply:
@andyw8 @tanema 

Commented code removed